test/controllers/api/v2/organizations_controller_test.rb

```diff
@@ -91,4 +92,77 @@ class Api::V2::OrganizationsControllerTest < ActionController::TestCase
     post :create, params: { :organization => { :name => "" } }
     assert_response :unprocessable_entity
   end
+
+  test "should not create with content type text plain" do
+    assert_raises_with_message(RuntimeError, 'Unknown Content-Type') do
+      post :create, params: { :organization => {:name => "foo organization"} }, as: 'text/plain'
+    end
+  end
+
+  test "create with name" do
+    name = RFauxFactory.gen_alpha
+    post :create, params: { :organization => {:name => name} }
+    assert_response :success
+    response = JSON.parse(@response.body)
+    assert response.key?('name')
+    assert_equal response['name'], name
+  end
+
+  test "create with name and description" do
+    name = RFauxFactory.gen_alpha
+    post :create, params: {:organization => { :name => name, :description => name } }
+    assert_response :success
+    result = JSON.parse(@response.body)
+    assert_equal result["name"], name
+    assert_equal result["description"], name
+  end
+
+  test "should not create with same name" do
+    name = Organization.first.name
+    post :create, params: { :organization => { :name => name} }
+    assert_response :unprocessable_entity
+    assert_include @response.body, "Name has already been taken"
+  end
+
+  test "search organization" do
+    organization = Organization.first
+    get :index, params: { :search =>  "name = \"#{organization.name}\"",  :format => 'json' }
+    assert_response :success
+    response = ActiveSupport::JSON.decode(@response.body)
```

Comment:
Any reason to use ```ActiveSupport::JSON``` over plain ```JSON``` here?

Reply:
@ekohl no any reason, done that by analogy found in other tests will update with the plain JSON